target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -25,6 +25,15 @@ avm,fritz1750e)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssi3" "wlan1" "60" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssi4" "wlan1" "80" "100"
 	;;
+avm,fritzdvbc)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:tv" "eth0"
```

Comment:
If it's labelled "tv", I don't think it should be used for lan?

Reply:
Since, for now, I can't get DVB-C working (requires internal USB working) the LED would otherwise have no use. Suggestions are welcome. Consider that users could easily remove the LAN LED setting in LuCI. But I guess you're right that it could be confusing. Would you prefer it unused by default?

Reply:
> Would you prefer it unused by default?

yes. LEDs meant for other purposes shouldn't be hijacked _by default_, but the default should resemble vendor behavior if possible. If somebody wants to use the tv led for LAN later, he/she could configure that manually afterwards.